cg/server/admin.py

```diff
@@ -441,6 +442,49 @@ class AnalysisView(BaseView):
     form_extra_fields = {"workflow": SelectEnumField(enum_class=Workflow)}
 
 
+class IlluminaFlowCellView(BaseView):
```

Comment:
```suggestion
class IlluminaSequencingRunView(BaseView):
```

Reply:
The flowcell is the run device model.

Reply:
The idea is that the view in the web API the table is called FlowCell, not Sequencing metrics

Reply:
resolved in person